docs/subnets/create-a-fuji-subnet.md

```diff
@@ -710,7 +587,12 @@ Use the arrow keys to navigate: ↓ ↑ → ←
 
 The default is `Yes` but just choose `No` here to speed up things, assuming the node is already whitelisted.
 
-There are now two choices possible: automatic and Manual configuration. As mentioned earlier,
+As mentioned earlier, If the node has not been whitelisted, a holder of the Subnet control key 
```

Comment:
We want to change what we suggest here, we should be advising that addValidator always be called AFTER join. Also capitalization of If.